MobileWallet/Screens/AppEntry/Splash/WalletCreationViewController.swift

```diff
@@ -462,19 +585,115 @@ class WalletCreationViewController: UIViewController {
     }
 
     private func showLocalAuthentification() {
-        self.createEmojiButton.animateIn()
+        createEmojiButton.animateIn()
+
+        let currentType = LAContext().biometricType
+        switch currentType {
+            case .faceID:
+                runFaceIDAnimation()
+            case .touchID:
+                runTouchIdAnimation()
+            case .none:
+                print("nothing")
+        }
+
+        secondLabelTopConstaint?.constant = -50
+
+        UIView.animate(withDuration: 0.5, animations: { [weak self] in
```

Comment:
What does this `animate` do? If nothing is being changed inside?

Reply:
This is how animate changing of constraints works.
1. secondLabelTopConstaint?.constant = -50 - i change the constraint value
2. Animate this in 0.5 seconds. -> self.view.layoutIfNeeded()

I did that because for the other label i want to move it a little later.